### PR TITLE
Add Reference Setup pages to main and 2.26

### PR DESCRIPTION
**Pull request by csengerszabo**

### CODE REVIEW

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+
+A helpful shortcut, could be our KubeOne tooling container, which contains all the needed tooling:
+
+<https://github.com/kubermatic/community-components/tree/master/container/kubeone-tool-container> 
+
+# **Load Balancer**
+
+Kubermatic exposes an NGINX server and user clusters API servers via Load Balancers. Therefore, KKP is using the native Kubernetes Service Type `LoadBalancer` implementation. More details about the different expose points you find at the next chapter “DHCP/Networking”
+
+\
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:69]: Remove typo

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+
+[![](kubelb.png)](https://docs.kubermatic.com/kubelb/v1.1/architecture/)
+
+As frontend IPAM solution and IP announcement, KubeLB could use on-premise non-multi-tenant LB implementations like Cilium or MetalLB in Layer 2 ARP or BGP mode. (Also commercial Kubernetes conform implementation like [F5 Big IP](https://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/v1.0/#) would work). KubeLB will add the multi-tenant plus central DNS, Certificate and Ingress management. KubeLB deliver for each Kubernetes Cluster one tenant separated authentication token, what get used via the so called [KubeLB CCM](https://docs.kubermatic.com/kubelb/v1.1/installation/tenant-cluster/), what automatically get configured for KKP clusters. The KubeLB CCM is then handling service and node announcements.  For Setups where multi-tenant automated LB is not required, direct [MetalLB](https://metallb.universe.tf/) or [Cilium](https://docs.cilium.io/) setups could be used as well. For the best performance and stability of the platform, we recommend to talk with our consultants to advise you what is the best fit for your environment.
+
+
+### ******Layer 2 ARP Announcement**
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:82]: Format broken see Preview

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 3]
```diff
@@ -0,0 +1,478 @@
...
+- [MetalLB BGP Configuration](https://metallb.io/configuration/_advanced_bgp_configuration/)
+
+- [Cilium BGP Control Plane](https://docs.cilium.io/en/stable/network/bgp-control-plane/bgp-control-plane/) 
+
+
+## **Public/Private Cloud Load Balancers**
+
+For other Load Balancer scenarios we strongly recommend to use cloud environment specific Load Balancer what comes with the dedicated cloud CCM. This native cloud LBs can interact dynamically with Kubernetes to provide updates for service type Load Balancer or Ingress objects. For more detail see [Kubernetes - Services, Load Balancing, and Networking](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer).
+
+\
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:104]: remove typo

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 4]
```diff
@@ -0,0 +1,478 @@
...
+
+- Direct Node-to-Node communication is needed on each cluster level (master/seed and user cluster level) based on generic Kubernetes requirements: [Kubernetes - Cluster Networking](https://kubernetes.io/docs/concepts/cluster-administration/networking/)
+
+- Check the more detailed requirements of the network at:
+
+  - [****KKP Documentation > Concepts > Networking****](https://docs.kubermatic.com/kubermatic/main/architecture/concept/kkp-concepts/networking) ****
+
+  - [****KKP Documentation > Networking > Control Plane Expose Strategy****](https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/networking/expose-strategies/)
+
+\
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:123]: remove typo

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 5]
```diff
@@ -0,0 +1,478 @@
...
+
+- DHCP for worker nodes is required
+
+- Verify what Load Balancing Options your target environment has, see section “Load Balancing”
+
+- Direct Node-to-Node communication is needed on each cluster level (master/seed and user cluster level) based on generic Kubernetes requirements: [Kubernetes - Cluster Networking](https://kubernetes.io/docs/concepts/cluster-administration/networking/)
+
+- Check the more detailed requirements of the network at:
+
+  - [****KKP Documentation > Concepts > Networking****](https://docs.kubermatic.com/kubermatic/main/architecture/concept/kkp-concepts/networking) ****
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:119]: check formating

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 6]
```diff
@@ -0,0 +1,478 @@
...
+
+
+# **DNS**
+
+Kubermatic requires DNS entries for NGINX and user clusters API servers. These have to be created after the Load Balancers, see [EE Installation - DNS Records](https://docs.kubermatic.com/kubermatic/main/installation/install-kkp-ce/#dns-records) and [EE Seed Cluster - DNS Records](https://docs.kubermatic.com/kubermatic/main/installation/install-kkp-ce/add-seed-cluster-ce/#dns-record).
+
+The NGINX server provides access to the Kubermatic UI and the logging and monitoring services (Prometheus, Kibana, etc.). It requires a DNS name like kubermatic.example.com and \*.kubermatic.example.com (e.g. for prometheus.kubermatic.example.com). Optional: if no wildcard DNS can be provided, dedicated DNS entries per exposed Service need to get entered (\~5 services).
+
+To access a user cluster via API, a wildcard DNS entry per seed cluster (in your case the master cluster is the only seed) has to be provided. E.g., \*.cluster-1.kubermatic.example.com. User clusters would be accessible via \[cluster-id].cluster-1.kubermatic.example.com. 
+
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:133]: check domain formating, better do it with  code  formating `dns-name`

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 7]
```diff
@@ -0,0 +1,478 @@
...
+| Management Monitoring - Alertmanager           | alertmanager.kubermatic.example.com                                                                                                                                                                                                                                                | Ingress IP: dynamic or static (virtual IP)                                                     |
+| Seed cluster - Kubernetes API Expose Service   | Expose Strategy - **NodePort**: \*.seed.kubermatic.example.comExpose Strategy - **LoadBalancer**: NONE(will be done by independent IPs per cluster see - [expose strategy LoadBalancer](https://docs.kubermatic.com/kubermatic/master/concepts/expose-strategy/expose_strategy/)). | NodePort Proxy: dynamic or static (virtual IP)**Or** one virtual IP per user cluster in a CIDR |
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:148]: Formatting and add Tunneling strategy

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 8]
```diff
@@ -0,0 +1,478 @@
...
+
+It is also possible to use [external-dns](https://github.com/kubernetes-sigs/external-dns) as automatic way to manage DNS Entries for you. Therefor checkout if your DNS provider is supported: [external-dns provider](https://github.com/kubernetes-sigs/external-dns?tab=readme-ov-file#externaldns)
+
+# **Certificates**
+
+Certificates are required for the Kubermatic UI, as well as logging and monitoring services. As automatic creation via cert-manager is not an option, please create in advance. According to the DNS example, this has to be valid for kubermatic.example.com, prometheus.kubermatic.example.com, grafana.kubermatic.example.com, alertmanager.kubermatic.example.com, and kibana.kubermatic.example.com. Our Helm charts require a single certificate file to be valid for these names (wildcard is fine). Any supported cert-manager automation (like e.g. let’s encrypt) could get configured out of the box, so please check  [Issuer Configuration - cert-manager Documentation](https://cert-manager.io/docs/configuration/)
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:156]: dns name formatting

File: content/kubermatic/main/references/reference_setup/_index.en.md [hunk 9]
```diff
@@ -0,0 +1,478 @@
...
+
+  - Workers <-> Seed communication is in the Azure Net only
+
+- User Cluster users need to reach the Azure seed cluster load balancer node-port
+
+  - IP/DNS of Azure LoadBalancer
+
+- Application traffic gets exposed to Azure workers (Cloud LBs managed by Azure CCM)
+
+- Host for Seed provisioning (KubeOne setup) needs to reach the Azure network VMs by SSH
```
Comment (toschneck) [inline content/kubermatic/main/references/reference_setup/_index.en.md:478]: we should split this in subpages or Tabs

File: content/kubermatic/main/references/reference_setup/_index.en.md (hunk 2 above)
Comment (csengerszabo) [reply content/kubermatic/main/references/reference_setup/_index.en.md]: Done

File: content/kubermatic/main/references/reference_setup/_index.en.md (hunk 3 above)
Comment (csengerszabo) [reply content/kubermatic/main/references/reference_setup/_index.en.md:104]: Done

File: content/kubermatic/main/references/reference_setup/_index.en.md (hunk 4 above)
Comment (csengerszabo) [reply content/kubermatic/main/references/reference_setup/_index.en.md:123]: Where? @toschneck

File: content/kubermatic/main/references/reference_setup/_index.en.md (hunk 6 above)
Comment (csengerszabo) [reply content/kubermatic/main/references/reference_setup/_index.en.md]: Done

File: content/kubermatic/main/references/reference_setup/_index.en.md (hunk 8 above)
Comment (csengerszabo) [reply content/kubermatic/main/references/reference_setup/_index.en.md:155]: Done

File: content/kubermatic/main/references/reference_setup/_index.en.md (hunk 9 above)
Comment (csengerszabo) [reply content/kubermatic/main/references/reference_setup/_index.en.md]: @toschneck Later can be an improvement, but let's not make this a blocker for merging this PR to the docs

Comment (toschneck) [reply content/kubermatic/main/references/reference_setup/_index.en.md]: this never get done then

**Comment by toschneck** [comment]
@csengerszabo  I adjusted formats and etc, please take a look at #1794. If this PR is merged, thise one will be merged as well.

**Comment by csengerszabo** [comment]
@toschneck can you approve this then?

**Comment by toschneck** [comment]
Ah as we sqetsch merge the PR we need to close this PR as my PR already contained the changes.

**Pull request closed without merging**